scripts/build.sh

```diff
@@ -41,55 +38,27 @@ rm -f bin/*
 rm -rf pkg/*
 mkdir -p bin/
 
-# If its dev mode, only build for our self
-if [ "${VAULT_DEV_BUILD}x" != "x" ]; then
-    XC_OS=$(go env GOOS)
-    XC_ARCH=$(go env GOARCH)
-    XC_OSARCH=$(go env GOOS)/$(go env GOARCH)
-fi
-
-
-# The main method we need for building is in the cmd directory
-cd "${DIR}/cmd/${TOOL}"
-
 # Build!
 echo "==> Building..."
-gox \
-    -osarch="${XC_OSARCH}" \
+echo "Running:"
+echo "${GO_CMD} build -gcflags ${GCFLAGS} -ldflags -X github.com/hashicorp/${TOOL}/version.GitCommit=${GIT_COMMIT}${GIT_DIRTY} -o bin/${TOOL} -tags ${BUILD_TAGS} ."
```

Comment:
Looks like we have 2 `go build` commands in this file

Reply:
oh, nevermind this is just `echo`-ing the command

Reply:
Oh thanks for catching that debug statement, I was having an issue with the build working initially but forgot to remove that after fixing the issue